apps/io-wallet-user-func/local.settings.json.example

```diff
@@ -29,7 +29,7 @@
     "PdvTokenizerApiBaseURL": "PDV_TOKENIZER_API_BASE_URL",
     "PdvTokenizerApiKey": "PDV_TOKENIZER_API_KEY",
     "PdvTokenizerTestUUID": "PDV_TOKENIZER_TEST_UUID",
-    "EntityConfigurationStorageAccount__serviceUri": "ENTITY_CONFIGURATION_STORAGE_ACCOUNT_SERVICE_URI"
+    "EntityConfigurationStorageAccount__serviceUri": "ENTITY_CONFIGURATION_STORAGE_ACCOUNT__SERVICE_URI",
```

Comment:
Why _ _ ?

Reply:
Just because there is `__` in the variable name 😄

Reply:
And `__serviceUri` suffix is mandatory for azure

Reply:
Ok, thanks